LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright 2014 Klaus Hartl
+Copyright 2006, 2014, 2015 Klaus Hartl & Fagner Brack
```

Comment:
Why do we need to specify the dates like `2006`, `2014` and `2015`? Can't we just put 2018?